ssp

```diff
@@ -8285,44 +8282,15 @@ sub check_for_newrelic {
 
 }
 
-sub check_for_zamfoo {
-    my $has_zamfoo = 0;
+sub check_for_multilevel_reseller {
+    my @ml_plugins = ( 'zamfoo', 'whmreseller', 'whmphp', 'whmamp' );
+    my $docroot = '/usr/local/cpanel/whostmgr/docroot/cgi';
```

Comment:
I'd like a different variable name here.  $docroot implies an absolute root RE: Apache so that's a bit misleading.